base_layer/wallet/src/util/emoji.rs

```diff
@@ -105,18 +105,21 @@ impl EmojiId {
 
     pub fn str_to_pubkey(s: &str) -> Result<PublicKey, ()> {
         let mut indices = Vec::with_capacity(33);
-        for c in s.chars() {
-            if let Some(i) = REVERSE_EMOJI.get(&c) {
-                indices.push(*i);
-            } else {
+        if !s.is_empty() {
```

Comment:
```suggestion
        if s.is_empty() { return Err(()); }
```

Why?
* Return early
* Don't nest

Reply:
Done